monero-outreach-docs/translations/zh-hans/guerrilla_marketing/guerrilla_marketing_zh-hans.md

```diff
@@ -0,0 +1 @@
这些是希望帮助提高门罗币的知名度的用户和投资者，但可能他们不喜欢采取激进的方式，或者影响力没有达到大使的级别。

Write posts on forums, and social media, and news sites to inform non-Monero users of the benefits of Monero and why they should be using it. A great example of this is when Monero’s very own /u/johnfoss69 posted on the large r/cryptocurrency sub-reddit: Your Guide to Monero, and Why It Has Great Potential. Get creative, though, because there are many other audiences outside of cryptocurrency enthusiasts whose lives might improve if they knew about Monero.
在论坛、社交媒体和新闻网站上发表文章，告诉非门罗币用户门罗币的好处，以及为什么他们应该使用它。一个很好的例子是门罗币自己的/u/johnfoss69发布子reddit r/cryptocurrency: 门罗币指南，以及为什么它有很大的潜力。要有创意，因为除了加密货币爱好者之外，还有许多其他观众，如果他们了解门罗币，他们的生活可能会有所改善。

Share Monero articles and resources with friends and followers on social media, even LinkedIn.
在社交媒体上，甚至是LinkedIn上与朋友和追随者分享门罗币的文章和资源。

Create or contribute to videos, podcasts, websites, etc and other online content.
创建或贡献视频、播客、网站等和其他在线内容。

Call-in to radio stations and advocate for Monero on-the-air.
打电话给电台，为门罗币做宣传。
Write an op-ed or letter to the editor to your local newspaper and talk about how Monero could be used to make your local economy more resilient to global financial risks.
给当地报纸的编辑写一篇专栏文章或一封信，谈谈如何利用门罗币使当地经济更能抵御全球金融风险。

Reach out to your local businesses and inform them of Monero, using toolkits available from Monero Outreach. We will continue to develop and improve our toolkits, so check-in regularly.
使用门罗币推广组提供的工具包，与当地企业联系，向他们介绍门罗币。我们将继续开发和改进我们的工具包，所以请经常查看。

Combine or add to any of these tactics in a way that you are comfortable with. See resources at the bottom.
以一种你觉得舒服的方式组合或添加这些策略。参见底部的参考资料。

Activists 活动积极分子
Social Media Activism. “High-jacking hashtags” is one way to get people’s attention. A simple DuckDuckGo search might yield you some more ideas as well.
社交媒体积极分子。“高质量的标签”是吸引人们注意的一种方式。一个简单的DuckDuckGo搜索可能会给你带来更多的想法。

Public Venues. Using public venues is a great way to leverage your time and energy. Think of concerts, fairs, major conferences, and political events, etc.
公共场所。利用公共场所是高效利用你的时间和精力的好方法。想想音乐会、博览会、大型会议和政治活动等。

Competitive Venues. 竞争场所
Using guerrilla marketing tactics at venues that would be considered competitors is another, a more risky, and controversial approach that can have a big impact. Think of financial institutions and the like.		
在被视为竞争对手的场所使用游击营销策略是另一种风险更大、争议更大的方法，可能会产生巨大影响。想想金融机构之类的。

Projection 投影
With an image, a small laptop, and a projector, you can make a big impact. Alternatively, you can use a bright flashlight and a cutout of the Monero logo in front of it. Think of Batman…	
有了一张图片、一台小型笔记本电脑和一台投影仪，你就能产生巨大的影响。或者，你也可以用一个明亮的手电筒，在它前面放一个门罗币的剪纸。想想蝙蝠侠……
	
Information Drop. Mass print information about Monero and place them around large venues to introduce Monero to a wide, or new, audience.
信息扩散。大量印刷关于门罗币的信息，并将其放置在大型场所周围，将门罗币介绍给广大的或新的观众。

QR Codes. Mass print QR codes with a short blurb that attracts attention and place them in around venue locations.
二维码。用简短的宣传语大量打印二维码，吸引人们的注意力，并将其放置在会场周围。



```

Comment:
- Move to folder _/zh-hans/website-sections_
- Change the files name to "_**guerrila_toolkit_zh-hans.md**_"

Reply:
Done.